### PR TITLE
Return track on subscribe

### PR DESCRIPTION
**Pull request by roznawsk**

### CODE REVIEW

File: rtsp/CHANGELOG.md
Comment (roznawsk) [inline rtsp/CHANGELOG.md:1]: no changes to RTSP in this PR, but changelog hasn't been updated previously

File: engine/lib/membrane_rtc_engine/engine.ex [hunk 1]
```diff
@@ -553,7 +553,14 @@ defmodule Membrane.RTC.Engine do
     case validate_subscription(subscription, ctx, state) do
       :ok ->
         {spec, state} = fulfill_or_postpone_subscription(subscription, ctx, state)
-        send(endpoint_pid, {ref, :ok})
+
+        track =
+          state.endpoints
+          |> Map.values()
+          |> Enum.flat_map(&Endpoint.get_tracks/1)
+          |> Enum.find(&(&1.id == track_id))
```
Comment (Karolk99) [inline engine/lib/membrane_rtc_engine/engine.ex:561]:
```suggestion
track = get_track(track_id, state.endpoints)
```

File: webrtc/lib/webrtc_endpoint.ex [hunk 2]
```diff
@@ -850,6 +842,18 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC do
...
+  defp subscribe_tracks(rtc_engine, endpoint_id, new_outbound_tracks) do
+    {valid, invalid} =
+      Enum.reduce(new_outbound_tracks, {[], []}, fn track, {valid, invalid} ->
+        case Engine.subscribe(rtc_engine, endpoint_id, track.id) do
+          {:ok, updated_track} -> {[updated_track | valid], invalid}
+          :ignored -> {valid, [track | invalid]}
+        end
+      end)
+
+    {Enum.reverse(valid), Enum.reverse(invalid)}
```
Comment (Karolk99) [inline webrtc/lib/webrtc_endpoint.ex:854]: I think the split_with was easier to read. And I'm not sure if order is relevant.

**Review by Karolk99 — APPROVED** [review]

File: engine/lib/membrane_rtc_engine/engine.ex (hunk 1 above)
Comment (roznawsk) [reply engine/lib/membrane_rtc_engine/engine.ex:561]: nice find

**Pull request merged**